FlagsmithClient/src/main/java/com/flagsmith/internal/FlagsmithRetrofitService.kt

```diff
@@ -49,31 +52,65 @@ interface FlagsmithRetrofitService {
                 }
             }
 
-            fun envKeyInterceptor(environmentKey: String): Interceptor {
+            fun jsonContentTypeInterceptor(): Interceptor {
                 return Interceptor { chain ->
-                    val request = chain.request().newBuilder()
-                        .addHeader("X-environment-key", environmentKey)
-                        .build()
-                    chain.proceed(request)
+                    val request = chain.request()
+                    if (chain.request().method == "POST" || chain.request().method == "PUT" || chain.request().method == "PATCH") {
+                        val newRequest = request.newBuilder()
+                            .header("Content-Type", "application/json; charset=utf-8")
+                            .header("Accept", "application/json")
+                            .build()
+                        chain.proceed(newRequest)
+                    } else {
+                        chain.proceed(request)
+                    }
+                }
+            }
+
+            fun updatedAtInterceptor(tracker: FlagsmithEventTimeTracker): Interceptor {
+                return Interceptor { chain ->
+                    val response = chain.proceed(chain.request())
+                    val updatedAtString = response.header("x-flagsmith-document-updated-at")
```

Comment:
Minor comment: this header string should probably be a constant defined somewhere?

Reply:
Sorted